index.html

```diff
@@ -24,164 +24,165 @@ <h1 id="title">Survey Form</h1>
+                <input id="zipCode" type="number" name="cepUsuario" required placeholder="Digite seu CEP">
+              </div> 
             </div>
-          </div>  
-          <div class="rowTab">
             <div class="labels">
-              <label for="comments">Gostaria de deixar algum comentário ou sugestão?</label>
+              <label for="dropdown">Qual dessas opções melhor descreve sua atual posição?</label>
             </div>
             <div class="rightTab">
-              <textarea id="comments" class="input-field" name="comment">Digite aqui...
-              </textarea>
+              <select id="dropdown" name="atualPosicaoUsuario" class="dropdown">
+                <option disabled value>Selecione uma opção</option>
+                <option value="student">Estudante</option>
+                <option value="job">Trabalho</option>
+                <option value="studyAndWork">Estudo e trabalho</option>
+                <option value="preferNo">Prefiro não dizer</option>
+                <option value="other">Outro</option>
+              </select>
             </div>
+            <div class="rowTab">
+              <div class="labels">
+                <label for="userRating">Você recomendaria nossos cursos para uma amiga?</label>
+              </div>
+              <div class="rightTab">
+                <ul>
+                  <li class="radio">
+                    <label>
+                      <input name="radio-buttons" value="1" type="radio" class="userRating">Com certeza 
+                    </label>
+                  </li>
+                  <li class="radio">
+                    <label>
+                      <input name="radio-buttons" value="2" type="radio" class="userRating">Talvez 
+                    </label>
+                  </li>
+                  <li class="radio">
+                    <label>
+                      <input name="radio-buttons" value="3" type="radio" class="userRating">Não 
+                    </label>
+                  </li>
+                </ul> 
+                <div class="labels">
+                  <label for="dropdownPreferences">O que você prefere em nossa plataforma?</label>
+                </div>
+                <div class="rightTab">
+                  <select id="dropdownPreferences" name="oQueUsuarioPrefere" class="dropdown">
+                    <option disabled value>Selecione uma opção</option>
+                    <option value="challenge">Desafios</option>
+                    <option value="project">Projetos</option>
+                    <option value="community">Comunidade</option>
+                    <option value="openSource">Open Source</option>
+                    <option value="other">Outro</option>
+                  </select>
+                </div>    
+              </div>
+            </div>  
+            <div class="rowTab">
+              <div class="labels">
+                <label for="improvedPreferences">
```

Comment:
Não precisa de label quando você tem multiplos campos. Como lista de checkbox ou input radios. Até porque seu `for` não vai funcionar. Os labels que já têm em cada elemento `<li>` já é o suficiente.